### DISCUSSION TITLE
Add to NGINX

### DISCUSSION THREAD

**Discussion by justason** [tteck/Proxmox#941] [Ideas]
Hello,
Jusrt add to the NGINX proxy documentation: In order to use with Homeassiatant - enable Websockets support in proxy host.
Otherwise cannot login from WAN

**Comment by proddy** [comment]
thanks @justason . I'm planning to setup nginx proxy tonight to get home assistant access externally, through my Unifi USG. And using Lets Encrypt cert. Got any tips or pointers I should be aware of?

**Reply by Vakrehus** [reply]
If you use Nginx Proxy Manager, add this to Advanced-Tab in Custom Nginx Configuration
```
proxy_set_header Upgrade $http_upgrade;
proxy_set_header Connection “upgrade
```
